### PR TITLE
updated type diagram for smaller screens

### LINKED ISSUE #374: Minor Update: Fix Lesson 2 Type Diagram for smaller screens
original:
image
For this one, let's actually have the diagrams readjust to be vertical instead.
updated:
image

### PR DESCRIPTION
**Pull request by EdmundYau**
## Summary

Closes #374 

- Changed TypeDiagram to be responsive to smaller screens

## Screenshots
image

### CODE REVIEW

File: src/styles/Lesson2.scss [hunk 1]
```diff
@@ -21,5 +21,15 @@
     display: flex;
     gap: 80px;
     justify-content: center;
+
+    /* sm */
+    @media (max-width: 375px) {
```
Comment (Dankco) [inline src/styles/Lesson2.scss]: Isn't this redundant with the other condition?

**Review by Dankco — APPROVED** [review]
I guess it can help with clarity anyway, so I am just going to merge. It is not that big of a deal to have a couple lines of redundant code. Nice small change!

**Comment by EdmundYau** [comment]
Oh, I removed it cause I thought it was redundant, I don't know if I should remove it or not.

**Comment by Dankco** [comment]
Yeah its fine to remove it. Doesn't matter too much.

**Pull request merged**